ETC.tsv

```diff
@@ -157,26 +157,26 @@ xETC_20150317_000156	Its slimy substance allows it to model itself into a protot
 ETC_20150317_000157	Rusrat
 xETC_20150317_000158	During the times of Zachariel the Great, these monsters were placed to stop intruders. Its name stems from grave robbers who encountered it.
 ETC_20150317_000159	Madakia
-xETC_20150317_000160	To fend off intruding grave robbers, court sorcerers created Madakia.
+xETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
 ETC_20150317_000161	Wheelen
 xETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
 ETC_20150317_000163	Wheelen 3
 ETC_20150317_000164	Mauros
-xETC_20150317_000165	The stones of the Royal Mausoleum form with magic into an ancient heavily armed knight.
+xETC_20150317_000165	The stones of the Royal Mausoleum magically assembled into an ancient, heavily armed knight.
 ETC_20150317_000166	Truffle
-xETC_20150317_000167	Though the reason is not known, this monster has an extreme hatred towards humans. It loves to attack them for food and survival.
+xETC_20150317_000167	This monster has an irrational and extreme hatred towards humans. It loves attacking them for food and survival.
```

Comment:
Keep original translation but change:
Though the reason is not known -> Though the reason is unknown
The original translation does not say anything about irrational. I'd leave that out because it could be rational, potentially.